src/lifetracker/calendar/CalendarEntry.java

```diff
@@ -3,6 +3,8 @@
 import java.time.LocalDateTime;
 import java.time.LocalTime;
 
+import lifetracker.calendar.CalendarEntryImpl.EntryType;
```

Comment:
Move the Enum to the interface.

Currently your Interface depends on the implementation, which defeats the purpose of the interface as a generalized abstraction.